cypress/e2e/shifting_spec/ShiftingRequest.cy.ts

```diff
@@ -0,0 +1,80 @@
+import LoginPage from "pageobject/Login/LoginPage"
+import { PatientConsultationPage } from "pageobject/Patient/PatientConsultation";
+import { PatientPage } from "pageobject/Patient/PatientCreation";
+import ShiftCreation from "pageobject/Shift/ShiftCreation";
+import { ShiftDetails } from "pageobject/Shift/ShiftDetails";
+import { ShiftingBoard } from "pageobject/Shift/ShiftingBoard";
+import ShiftUpdate from "pageobject/Shift/ShiftUpdate";
+
+describe("New shifting request",() =>{
+  const loginPage = new LoginPage();
+  const shiftCreation = new ShiftCreation()
+  const shiftingBoard = new ShiftingBoard()
+  const shiftDetails = new ShiftDetails()
+  const shiftUpdate = new ShiftUpdate()
+  const patientPage = new PatientPage()
+  const patientConsultationPage = new PatientConsultationPage() 
+
+  before(() => { 
+    loginPage.loginAsDistrictAdmin(); 
+    cy.saveLocalStorage(); 
+  });
+
+  beforeEach(() => { 
+    cy.restoreLocalStorage(); 
+  });
+
+  it("Creating and verifying new shifting request", () => {
+    cy.intercept("POST", "api/v1/shift").as("shiftRequest"); 
```

Comment:
- this intercept should be a function
- it should be placed before submitting the form